test/server/admin/prometheus_stats_test.cc

```diff
@@ -57,6 +59,16 @@ class PrometheusStatsFormatterTest : public testing::Test {
                                        Stats::Gauge::ImportMode::Accumulate));
   }
 
+  void addTextReadout(const std::string& name, const std::string& value,
+                      Stats::StatNameTagVector cluster_tags) {
+    Stats::StatNameManagedStorage name_storage(baseName(name, cluster_tags), *symbol_table_);
+    Stats::StatNameManagedStorage tag_extracted_name_storage(name, *symbol_table_);
+    auto textReadout = alloc_.makeTextReadout(name_storage.statName(),
```

Comment:
explicit type here (not obvious from context).

Reply:
Done